Change the path for the databases to /var/cache/command-not-found
--- a/command-not-found
+++ b/command-not-found
@@ -76,7 +76,7 @@ def main():
         version=__version__,
         usage=_("%prog [options] <command-name>"))
     parser.add_option('-d', '--data-dir', action='store',
-                      default="/usr/share/command-not-found",
+                      default="/var/cache/command-not-found",
                       help=_("use this path to locate data fields"))
     parser.add_option('--ignore-installed', '--ignore-installed',
                       action='store_true',  default=False,
--- a/CommandNotFound/CommandNotFound.py
+++ b/CommandNotFound/CommandNotFound.py
@@ -74,7 +74,7 @@ class CommandNotFound(object):
 
     output_fd = sys.stderr
 
-    def __init__(self, data_dir="/usr/share/command-not-found"):
+    def __init__(self, data_dir="/var/cache/command-not-found"):
         self.sources_list = self._getSourcesList()
         # a new style DB means we can skip loading the old legacy static DB
         if os.path.exists(dbpath):
